subxt/Cargo.toml

```diff
@@ -13,11 +13,17 @@ description = "Submit extrinsics (transactions) to a substrate node via RPC"
 keywords = ["parity", "substrate", "blockchain"]
 
 [features]
+default = ["jsonrpsee"]
+
 # Activate this to expose functionality only used for integration testing.
 # The exposed functionality is subject to breaking changes at any point,
 # and should not be relied upon.
 integration-tests = []
 
+# Jsonrpsee if the default RPC provider used in Subxt. However, it can be
+# swapped out for an alternative implementation, and so is optional.
+jsonrpsee = ["dep:jsonrpsee"]
```

Comment:
why is this feature needed at all?

`jsonrpsee` is already an optional dep?

Reply:
I literally just added it so that there was a place to add a comment about the feature above it (and also I quite like when all of the features are clearly listed in one place in the file, rather than needing to spot which dependencies are optional; it feels a bit clearer to me :)